pyproject.toml

```diff
@@ -119,8 +125,10 @@ exclude_also = [
 
 [tool.pytest.ini_options]
 addopts = [
+    # "--import-mode=importlib",  # TODO: enable
```

Comment:
Some explanation of why, link to a relevant issue here would be good

Reply:
because you said so, there’s no issue that I’m aware of